<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Quit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>MIDI Devices</string>
    </property>
    <widget class="QLabel" name="label_File">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>114</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QComboBox" name="InputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>371</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="OutputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>371</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ouput">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>74</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouput</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FileCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>371</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatusInput">
     <property name="geometry">
      <rect>
       <x>449</x>
       <y>35</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LED</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatusOuput">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>74</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LED</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatusMidifile">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>113</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LED</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>MIDI Channels</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
    <widget class="QPushButton" name="pushButton_TracksAll">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>97</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_TracksFirst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_TracksNone">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>30</y>
       <width>97</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Panic">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>380</y>
      <width>97</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Panic</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>MIDI Controller</string>
    </property>
    <widget class="QPushButton" name="pushButton_FileIndex">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>30</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Midifile</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Speed">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>30</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Humanize">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Humanize</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Mode">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>30</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Playback</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Info">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>More</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
